### PR TITLE
9.0.2: Changelog

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

* Changelog for the next point release.

#### Jetpack product discussion

* N/A 

#### Does this pull request change what data or activity we track or use?

* N/A 

#### Testing instructions:

* Check for typos

#### Proposed changelog entry for your changes:

* This is it.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -243,6 +243,23 @@ Our Cookie and Consent Banner can help you comply with GDPR. The European Union
...
+**Enhancements**
+
+* Publicize: improve handling of URLs when generating Twitter threads.
+
+**Bug fixes**
+
+* Instagram Embeds: ensure that Instagram URLs with additional URL parameters can be embedded as well.
+* Media Extractor: prevent PHP notice for some URLs.
+* Publicize: avoid block editor errors when no Publicize connections are available.
+* Synchronization flow: avoid potential Fatal errors when updating the plugin from the Dashboard > Updates screen.
```
Comment (anomiex) [inline readme.txt:259]: It strikes me as odd that "Fatal" is capitalized here.

Comment (jeherve) [reply readme.txt:259]: Updated in 069442665cd3f5b5c3ad1aece9f52821d3e610a8

**Review by anomiex — APPROVED** [review]
If there are more typos, I'm not seeing them. (:

**Review by fgiannar — APPROVED** [review]
👍

**Comment by jeherve** [comment]
Cherry-picked to `branch-9.0` in f79170ec19b62d6fde9ec9579658a027ce694f0e

**Pull request merged**